utils/clients/tenderly.ts

```diff
@@ -117,40 +133,68 @@ async function simulateNew(config: SimulationConfigNew): Promise<SimulationResul
     timelockStorageObj[`queuedTransactions[${hash}]`] = 'true'
   })
 
+  if (governorType === 'oz') {
+    const id = hashOperationBatchOz(targets, values, calldatas, HashZero, keccak256(toUtf8Bytes(description)))
+    timelockStorageObj[`_timestamps[${id.toHexString()}]`] = simTimestamp.toString()
+  }
+
   // Use the Tenderly API to get the encoded state overrides for governor storage
-  const proposalKey = `proposals[${proposalId.toString()}]`
+  let governorStateOverrides: Record<string, string> = {}
+  if (governorType === 'bravo') {
+    const proposalKey = `proposals[${proposalId.toString()}]`
+    governorStateOverrides = {
+      proposalCount: proposalId.toString(),
+      [`${proposalKey}.id`]: proposalId.toString(),
+      [`${proposalKey}.proposer`]: DEFAULT_FROM,
+      [`${proposalKey}.eta`]: eta.toString(),
+      [`${proposalKey}.startBlock`]: proposal.startBlock.toString(),
+      [`${proposalKey}.endBlock`]: proposal.endBlock.toString(),
+      [`${proposalKey}.canceled`]: 'false',
+      [`${proposalKey}.executed`]: 'false',
+      [`${proposalKey}.forVotes`]: votingTokenSupply.toString(),
+      [`${proposalKey}.againstVotes`]: '0',
+      [`${proposalKey}.abstainVotes`]: '0',
+    }
+
+    targets.forEach((target, i) => {
+      const value = BigNumber.from(values[i]).toString()
+      governorStateOverrides[`${proposalKey}.targets[${i}]`] = target
+      governorStateOverrides[`${proposalKey}.values[${i}]`] = value
+      governorStateOverrides[`${proposalKey}.signatures[${i}]`] = signatures[i]
+      governorStateOverrides[`${proposalKey}.calldatas[${i}]`] = calldatas[i]
+    })
+  } else if (governorType === 'oz') {
+    const proposalCoreKey = `_proposals[${proposalId.toString()}]`
+    const proposalVotesKey = `_proposalVotes[${proposalId.toString()}]`
+    governorStateOverrides = {
+      [`${proposalCoreKey}.voteEnd._deadline`]: simBlock.sub(1).toString(),
+      [`${proposalCoreKey}.canceled`]: 'false',
+      [`${proposalCoreKey}.executed`]: 'false',
+      [`${proposalVotesKey}.forVotes`]: votingTokenSupply.toString(),
+      [`${proposalVotesKey}.againstVotes`]: '0',
+      [`${proposalVotesKey}.abstainVotes`]: '0',
+    }
+
+    targets.forEach((target, i) => {
+      const id = hashOperationOz(target, values[i], calldatas[i], HashZero, HashZero)
+      governorStateOverrides[`_timestamps[${id}]`] = '2' // must be > 1.
```

Comment:
nit: confused about the motivation of this line

Reply:
It's to pass this `timestamp > _DONE_TIMESTAMP` check passes

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/3d7a93876a2e5e1d7fe29b5a0e96e222afdc4cfa/contracts/governance/TimelockController.sol#L156

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/3d7a93876a2e5e1d7fe29b5a0e96e222afdc4cfa/contracts/governance/TimelockController.sol#L31